src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -657,10 +657,9 @@ TEST_P(object_data_handler_test_t, multiple_remap) {
   run_async([this] {
     multiple_write();
     auto pins = get_mappings(0, 128<<10);
-    EXPECT_EQ(pins.size(), 10);
+    EXPECT_EQ(pins.size(), 3);
```

Comment:
I believe that the existing test case might not be accurate against this commit.
In the multiple_remap test case, a extent (0 ~ 128 << 10) is initially written. Subsequently, five extents with distinct ranges are overwritten. So, in the existing case, the assert is correct if there are five mappings after the transaction is done. But, it is not correct with this commit because four writes after the initial write are merged into the extent currently being processed---please note that the initial write generates three mapping ranges. This results in only three mapping ranges remaining after the transaction is done.

Reply:
Hm, interesting to see that the adjustment is reflected in the UT.

I think the new mapping result is more approriate than the previous one according to my another comment "overwriting a pending extent shouldn't involve remapping".